### PR TITLE
New Metric Template: Update openssf-best-practices.md

### PR DESCRIPTION
**Pull request by yigakpoa**
Two things i noticed while working on updating this metric:

The phrase “The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see CHAOSS Data Ethics document for additional guidance. See OpenSSF’s API documentation for relevant information.” was put under implementation, so i moved it to additional information as it’s not a way to implement this metric.

Secondly, our Augur link doesn’t work or isn’t loading from my end: http://augur.osshealth.io/repo/Zephyr-RTOS/zephyr/risk

### CODE REVIEW

File: focus-areas/security/openssf-best-practices.md [hunk 1]
```diff
@@ -1,41 +1,67 @@
...
-Consumers of the badge can quickly assess which open source projects are following best practices and as a result are more likely to produce higher-quality secure software.
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
 
-## Implementation
-*The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see [CHAOSS Data Ethics document](https://github.com/chaoss/community/blob/main/data-use-statement.md) for additional guidance.*
-
-See [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.
+### Data Collection Strategies
+As gotten from the [OpenSSF Best Practices Badging Page](https://bestpractices.coreinfrastructure.org/en), a project’s OpenSSF Best Practices status can be assessed when:
```
Comment (ElizabethN) [inline focus-areas/security/openssf-best-practices.md:21]: Can you remove the word "gotten" and change the URL to this: https://www.bestpractices.dev/en. Thank you!

File: focus-areas/security/openssf-best-practices.md [hunk 2]
```diff
@@ -1,41 +1,67 @@
...
+- Projects receive a passing badge if they meet all required criteria.
+- The status of each criterion, for a given project, can be 'Met', 'Unmet', 'N/A' or 'unknown'.
+- Each criterion is in one of four categories: 'MUST', 'SHOULD', 'SUGGESTED', or 'FUTURE'.
+- To obtain a badge, all the MUST and MUST NOT criteria must be met, all SHOULD criteria must be met OR the rationale for not implementing the criterion must be documented, and all SUGGESTED criteria have to be rated as met or unmet.
+- Advanced badge levels of silver and gold are available if the project satisfies the additional criterion.
+
+For further information, refer to [OpenSSF’s API documentation](https://bestpractices.coreinfrastructure.org/en)
```
Comment (ElizabethN) [inline focus-areas/security/openssf-best-practices.md:28]: Can you change this URL to https://www.bestpractices.dev/en? Thank you!

File: focus-areas/security/openssf-best-practices.md [hunk 3]
```diff
@@ -1,41 +1,67 @@
...
+- Matt Germonprez
+- Daniel Izquierdo
+- Dawn Foster
+- Beth Hancock
+- Kevin Lumbard
+- Vinod Ahuja
+- Yigakpoa L. Samuel (Ikpae)
+
+## Additional Information
+*The usage and dissemination of health metrics may lead to privacy violations. Organizations may be exposed to risks. These risks may flow from compliance with the GDPR in the EU, with state law in the US, or with other law. There may also be contractual risks flowing from terms of service for data providers such as GitHub and GitLab. The usage of metrics must be examined for risk and potential data ethics problems. Please see [CHAOSS Data Ethics document](https://github.com/chaoss/community/blob/main/data-use-statement.md) for additional guidance, and [OpenSSF’s API documentation](https://github.com/coreinfrastructure/best-practices-badge/blob/master/doc/api.md) for relevant information.*
```
Comment (ElizabethN) [inline focus-areas/security/openssf-best-practices.md:59]: We actually add this as a separate module in the Wordpress page so you can delete this paragraph entirely. Thanks!

**Review by ElizabethN — CHANGES_REQUESTED** [review]
Please see the inline comments for requested changes. Thank you so much!

File: focus-areas/security/openssf-best-practices.md (hunk 1 above)
Comment (yigakpoa) [reply focus-areas/security/openssf-best-practices.md]: Done ✔️

File: focus-areas/security/openssf-best-practices.md (hunk 2 above)
Comment (yigakpoa) [reply focus-areas/security/openssf-best-practices.md]: Done, thank you

File: focus-areas/security/openssf-best-practices.md (hunk 3 above)
Comment (yigakpoa) [reply focus-areas/security/openssf-best-practices.md]: Done, thank you

**Pull request merged**